session.rst

```diff
@@ -3,6 +3,11 @@ Sessions
 
 .. toctree::
     :maxdepth: 1
-    :glob:
 
-    session/*
+    session/sessions_directory
+    session/avoid_session_start
```

Comment:
@sroze not sure if you're the best person, but I was wondering if the above article needs to change at all for the new lazy session handling stuff?

Reply:
Erm, that's a good point. I'll have a look.